packages/shared/src/convert-to-className.js

```diff
@@ -29,7 +29,7 @@ export function convertStyleToClassName(
   atRules: $ReadOnlyArray<string>,
   options: StyleXOptions = defaultOptions,
 ): StyleRule {
-  const { classNamePrefix = 'x' } = options;
+  const { classNamePrefix = 'x', dev = false } = options;
```

Comment:
We should add a new config option for this ability.
The new option should be optional and should default to the value of `dev`.

Reply:
I disagree. This should be the default for dev, and replace the other dev class names

Reply:
I think the question on whether we want to add an override has some considerations around:
1. Do we want users to be able to disable dev classnames? ie. is there a use case where users may want to test with the same classnames as in production? I think @nmn mentioned there may be some debug cases where this may be useful, will wait for weigh in here
2. Are we opinionated in allowing/restricting this in production as well? 

In either case, having an override could be helpful. Otherwise agree with mapping to dev value.

Reply:
If we want to separate "dev" and "debug" capabilities, we can change the human-readable name to be generated when a "debug" field is `true` instead. RSD makes this kind of distinction too.

Reply:
let's go with this approach then - we'll prefix the className in "debug" mode via a new config and have the dev/prod behaviour remain the same

Reply:
Yes. `debug` is a far better name. Let's go with that!